Adapt DateTimePicker styling on mobile devices (#3765)
vivid-planet/comet: .changeset/old-oranges-applaud.md

---
"@comet/admin": patch
---

> The placeholder in the `ClearInputAdornment` is no longer overlapped by a non-visible clear-button

Reviewer: The issue is caused by ClearInputAdornment, but it has no placeholder. So I'd suggest:
```suggestion
Prevent overlapping placeholders by a non-visible clear-button
```
